admin/client/EditorCustomizeTab.tsx

```diff
@@ -37,30 +49,104 @@ class ColorSchemeSelector extends React.Component<{ chart: ChartConfig }> {
         this.props.chart.props.invertColorScheme = value || undefined
     }
 
+    createLinearGradient(colorScheme: ColorScheme, count: number) {
+        let colors = colorScheme.getColors(count)
+        if (this.props.chart.props.invertColorScheme) colors = colors.reverse()
```

Comment:
If the color scheme is inverted, the preview also reverses.
I'm not sure whether we should introduce a new prop `invertedColorScheme` right here in `ColorSchemeSelector`, reflecting `this.props.chart.props.invertColorScheme`?